tools/bro-to-vast/bro-to-vast.cpp

```diff
@@ -81,7 +84,8 @@ class config : public broker::configuration {
     // As a stand-alone application, we reuse the global option group from CAF
     // to avoid unnecessary prefixing.
     opt_group{custom_options_, "global"}
-      .add<uint16_t>("port,p", "the port to listen at or connect to")
+      .add<std::string>("address,a", "the address where the endpoints listens")
```

Comment:
We could also use an URI here instead of having port and address separately configured.